snmp/datadog_checks/snmp/compat.py

```diff
@@ -1,21 +1,29 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from typing import Any, Optional
+
 try:
     from datadog_checks.base.utils.common import total_time_to_temporal_percent
 except ImportError:
 
     # Provide fallback for agent < 6.16
-    def total_time_to_temporal_percent(total_time, scale=1000):
+    def total_time_to_temporal_percent(total_time, scale=1000):  # type: ignore
```

Comment:
Note: mypy complains with "function overload must have same signature", so we can't add types for this until base `total_time_to_temporal_percent` gets types. Hence `# type: ignore`.